spec/features/account_manager/deleting_an_entry_spec.rb

```diff
@@ -1,11 +1,9 @@
 require 'spec_helper'
 
-feature 'account manager deleting an entry' do
+feature 'deleting an entry' do
```

Comment:
If the role name was part of the spec description, failing specs would provide better information.